src/v2/view-builder/ViewFactory.js

```diff
@@ -97,6 +102,15 @@ const VIEWS_MAPPING = {
     'security_question': EnrollAuthenticatorSecurityQuestion,
     email: EnrollAuthenticatorEmailView
   },
+  [RemediationForms.ENROLL_POLL]: {
+    app: EnrollAuthenticatorOVPollView,
```

Comment:
`EnrollPollOktaVerifyView` ?
otherwise would suggest to change form name.. `enroll-authenticator-poll` then name view `EnrollAuthenticatorPollOktaVerifyView`